src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -31,6 +32,7 @@ def osd_task(name, metadata, wait_for=2.0):
 
 
 @ApiController('/osd', Scope.OSD)
+@ControllerDoc('Osd management API', 'Osd')
```

Comment:
```suggestion
@ControllerDoc('OSD management API', 'OSD')
```
All-caps: OSD is the acronym for Object Storage Daemon.